nearcore/src/config.rs

```diff
@@ -816,13 +817,23 @@ pub fn init_configs(
     download_config_url: Option<&str>,
     boot_nodes: Option<&str>,
     max_gas_burnt_view: Option<Gas>,
-) {
+) -> Result<()> {
```

Comment:
```suggestion
) -> anyhow::Result<()> {
```

we have many different results, so being explicit helps